### PR TITLE
Upgrade to v1.12.0 of the CloudAMQP Terraform Provider

### PR DESCRIPTION
**Pull request by jkodroff**

### CODE REVIEW

**Comment by jkodroff** [comment]
@stack72 While building, I noticed that there's a mismatch on the docs page for integration metrics.  Note how the resource name is singular and the docs page is pluralized: https://registry.terraform.io/providers/cloudamqp/cloudamqp/latest/docs/resources/integration_metrics

Should we submit a PR upstream to fix that issue?  If it's possible to override within Pulumi, I'm not sure how.

File: provider/resources.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func refProviderLicense(license tfbridge.TFProviderLicense) *tfbridge.TFProvider
 }
 
 func Provider() tfbridge.ProviderInfo {
-	p := shimv1.NewProvider(cloudamqp.Provider())
+	p := shimv1.NewProvider(cloudamqp.Provider(""))
```
Comment (jkodroff) [inline provider/resources.go:68]: Also not sure if this is the right value to pass to the provider constructor.

**Comment by jkodroff** [comment]
Fixes #87 
Fixes #89

**Review by stack72 — APPROVED** [review]

File: provider/resources.go (hunk 1 above)
Comment (stack72) [reply provider/resources.go:68]: This is fine - this is just to satisfy the constructor for the upstream Provider

**Comment by stack72** [comment]
> @stack72 While building, I noticed that there's a mismatch on the docs page for integration metrics. Note how the resource name is singular and the docs page is pluralized: [registry.terraform.io/providers/cloudamqp/cloudamqp/latest/docs/resources/integration_metrics](https://registry.terraform.io/providers/cloudamqp/cloudamqp/latest/docs/resources/integration_metrics)
> 
> Should we submit a PR upstream to fix that issue? If it's possible to override within Pulumi, I'm not sure how.

This is kinda a bug in how *we* generate our docs :/ There will be a set of work in Q1 for this

**Pull request merged**